WatchParty/client/src/components/MoviePage/MoviePage.jsx

```diff
@@ -56,7 +56,7 @@ class MoviePage extends React.Component {
           <img
             className="posters"
             data-testid="poster-img"
-            onError={this.defaultPoster}
+            onError={(event) => { event.target.src = "/default.png"; }}
```

Comment:
Just a nit, it's usually better to have this call a function instead of writing it inline (for future)